content/applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst

```diff
@@ -16,35 +16,22 @@ ease the work of your cashiers.
 Configuration
 =============
 
-Configure the Payment Method
+Configure the payment method
 ----------------------------
 
-First, go in the general settings of the POS app, and activate the
-Vantiv setting.
+Enable the payment terminal :ref:`in the application settings <configuration/settings>` and
+:doc:`create the related payment method <../../payment_methods>`. Set the journal type as
+:guilabel:`Bank` and select :guilabel:`Vantiv` in the :guilabel:`Use a Payment Terminal` field.
+Then, type the name you want to give to your :guilabel:`Vantiv Credentials` and click
+:guilabel:`Create and edit`. Enter your :guilabel:`Merchant ID` and :guilabel:`Merchant Password`,
+then click :guilabel:`Save & Close`.
 
-.. image:: vantiv/vantiv_01.png
-    :align: center
+.. image:: vantiv/vantiv-method.png
+   :alt: Vantiv payment method
```

Comment:
The problem here is the displayed image is in master, but the target is 15.0. When it's only slight changes in the UI, we can oc take screenshots from master, but here, the changes are too great and do not fit the text (it shows fields that don't exist in 15.0)
I think we should have a 15.0 screenshot here, and come back to update it in the fx ports (don't worry, Audrey knows and will help you tomorrow)
Also, I think it's fine to show "create & edit without a terribly beek "-->"
I suggest this: 
![image](https://github.com/odoo/documentation/assets/95279628/a02ee9a3-f9a5-443b-934f-f37316143fc1)
(There's no outstanding account because I'm on a base database; less useless information. Here's how to reach a base db https://scribehow.com/shared/Connect_Odoo_Runbot_to_a_base__WCyRmAN1QYaEWVrNvlRG0g - note that normally no app is installed, which is why there's less information)